chart/flux/templates/helm-operator-deployment.yaml

```diff
@@ -118,6 +118,7 @@ spec:
           mountPath: /var/fluxd/helm/repository/cache
         {{- end }}
         args:
+        - --log-format={{ .Values.logFormat }}
```

Comment:
This should be set only if `.Values.logFormat` exists otherwise this introduces a breaking change.